packages/contracts-bedrock/test/kontrol/README.md

```diff
@@ -190,3 +190,67 @@ Therefore, the snapshots CI check will fail if the committed Kontrol state diff
 Note that the CI check only compares the JSON state diff, not the generated `DeploymentSummary.sol` or `DeploymentSummaryCode` contracts.
 This is for simplicity, as those three files will be in sync upon successful execution of the `make-summary-deployment.sh` script.
 We commit the `DeploymentSummary.sol` and `DeploymentSummaryCode.sol` contracts, because forge fails to build if those contracts are not present—it is simpler to commit these autogenerated files than to workaround their absence in forge builds.
+
+### KaaS (Kontrol as a Service) CI Integration
+
+KaaS (Kontrol as a Service) is a service provided by Runtime Verification Inc. It is used to execute Kontrol proofs in a CI pipeline. RV offers a hardware cluster to run these proofs, and the results are posted back to the commit statuses for the corresponding run request. GitHub statuses report a pass/fail status back to a commit, and each status result can be viewed by following the "Details" link. In a pull request, this status and link can be found under the PR Checks table at the bottom.
+
+To use KaaS, you need a Runtime Verification assigned token that provides access to run compute jobs under the RV Org. Additionally, if you want to receive status updates on the pass/fail of compute executions, you must provide a Fine-Grained token with the necessary statuses permissions to RV.
```

Comment:
Can we tweak the phrasing a bit to make it sound less like RV docs and more specific to our use case, e.g. "we have a Runtime Verification assigned token" :)

> Additionally, if you want to receive status updates on the pass/fail of compute executions, you must provide a Fine-Grained token with the necessary statuses permissions to RV.

What does this part mean? What are "compute executions" compared to the proof results?